sdks/java/container/java11/build.gradle

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id 'base'
+    id 'org.apache.beam.module'
+}
+
+project.ext {
+    imageJavaVersion = '11'
```

Comment:
My dream, which may be impossible, is that this can be passed as a parameter somehow to the `apply from:` line so that the control and data flow are more clear.

Reply:
I don't think apply from supports this (I think? gradle documentation is confusing but it seems like it's not supported via https://docs.gradle.org/current/dsl/org.gradle.api.Project.html#org.gradle.api.Project:apply(java.util.Map)) but I tried to make this slightly cleaner so these files literally only holds the parameters before applying the script